source/includes/fundamentals/code-examples/specify-query/FindAndBuilder.cs

```diff
@@ -0,0 +1,8 @@
+var builder = Builders<Guitar>.Filter;
+var filter = builder.And(builder.Gte(g => g.EstablishedYear, 1985), builder.Ne(r => r.Make, "Kiesel"));
```

Comment:
I would demonstrate the cool support for C# operators:
`var filter = builder.Gte(g => g.EstablishedYear, 1985) & builder.Ne(r => r.Make, "Kiesel")`.
Not crucial.

Reply:
I think this would be nice to show somewhere, but maybe not here since this code is intended to show the `And()` method specifically